.github/workflows/ci-docs.yaml

```diff
@@ -1,3 +1,6 @@
+# This file is created to address the issue described in
```

Comment:
Why do we need to link a stack overflow issue here? It's coming from GitHub's own documents. I think that would suffice.

Reply:
@davidhsingyuchen for context: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks

Reply:
@weikequ Thanks a lot for the context! @ahsan-z-khan Fixed in [48e0498](https://github.com/runfinch/finch/pull/28/commits/48e0498635d9c3fb5c28de573c8ec82733987a77).

Reply:
Resolving this thread due to https://github.com/runfinch/finch/pull/28#pullrequestreview-1190879386.